.github/workflows/danger.yml

```diff
@@ -0,0 +1,15 @@
+name: Rake on PR
+on: [push, pull_request]
```

Comment:
The PR linter should only run on `pull_request`, remove push or it will error later.